sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -652,6 +708,8 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
   struct FileSystemListPathsResult
   {
+    Azure::Core::Nullable<std::string> ETag;
+    Azure::Core::Nullable<std::string> LastModified;
```

Comment:
Why nullable?

Reply:
Talked offline, server behavior.

Reply:
If the server never returns ETag, then this field should not exist at all.
If the server always returns ETag, this filed should not be nullable.
If the server returns ETag in some cases while doesn't return in other cases, this field should be nullable.

We only talked about when the server doesn't return this header. You still need to explain when the server returns this header.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/1637f4b9ed998c40d9ae3fe054de3382b4769150/sdk/storage/Azure.Storage.Files.DataLake/src/Generated/DataLakeRestClient.cs#L1016
I don't know when the server returns these headers but .NET SDK does have them in response object so I don't think it is far off for C++ SDK to have them.

Reply:
as we've talked offline, remove these headers and create an issue @katmsft